.github/workflows/clippy.yml

```diff
@@ -0,0 +1,26 @@
+on: [push]
+
+name: Clippy
```

Comment:
Hum also: do you need all these dependencies to run clippy?

Reply:
For Clippy, not necessarily, no. I'll remove that.